src/main/java/org/sonar/plugins/csharp/CSharpSensor.java

```diff
@@ -285,6 +291,13 @@ private void handleRoslynIssue(String repositoryKey, String ruleId, String uri,
     }
   }
 
+  private static void handleRoslynProjectIssue(Issuable projectIssuable, String repositoryKey, String ruleId, String fullMessage) {
+      IssueBuilder builder = projectIssuable.newIssueBuilder();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)